lib/kernel/sync.rb

```diff
@@ -25,7 +25,7 @@ def Sync(&block)
 			reactor = Async::Reactor.new
 			
 			begin
-				return reactor.run(finished: ::Async::Condition.new, &block).wait
+				return reactor.run(*arguments, **options, finished: ::Async::Condition.new, &block).wait
```

Comment:
I don't think we should pass through all options as in some cases they won't be used... it would be confusing if some cases they have an effect and other cases they don't.

Reply:
I've addressed that in c166dcc